css/properties/text-decoration-line.json

```diff
@@ -151,7 +151,7 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "8"
```

Comment:
Tested [this example](http://bcd-examples.gh.caugner.de/css/text-decoration-line/) in Safari 5, 6 and 7 via BrowserStack Live, and it appears to have already been supported.

```suggestion
                "version_added": "6"
```

Safari 6.2.8:

<img width="781" alt="image" src="https://github.com/user-attachments/assets/fd0ad754-2e71-4438-910b-010fbbdc8ade">

Safari 5.1.5:

<img width="781" alt="image" src="https://github.com/user-attachments/assets/655a3d71-b0c0-4577-a6a5-2673d36a5aee">

Reply:
Oh!  No, Safari 6.2 is a backport of Safari 8; see https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#backported-releases for more details on that.  BrowserStack _does_ have Safari 6.0, and that should always be used for testing instead of 6.2!